awx/ui_next/src/screens/ManagementJob/ManagementJob.jsx

```diff
@@ -0,0 +1,154 @@
+  Redirect,
+  Route,
+  Switch,
+  useLocation,
+  useParams,
+  useRouteMatch,
+} from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { CaretLeftIcon } from '@patternfly/react-icons';
+import { Card, PageSection } from '@patternfly/react-core';
+
+import { SystemJobTemplatesAPI } from '../../api';
+import ContentError from '../../components/ContentError';
+import ContentLoading from '../../components/ContentLoading';
+import NotificationList from '../../components/NotificationList';
+import RoutedTabs from '../../components/RoutedTabs';
+import { Schedules } from '../../components/Schedule';
+import { useConfig } from '../../contexts/Config';
+import useRequest from '../../util/useRequest';
+
+function ManagementJob({ i18n, setBreadcrumb }) {
+  const basePath = '/management_jobs';
+
+  const match = useRouteMatch();
+  const { id } = useParams();
+  const { pathname } = useLocation();
+  const { me, isNotificationAdmin } = useConfig();
+
+  const canReadNotifications = isNotificationAdmin || me?.is_system_auditor;
+
+  const { isLoading, error, request, result } = useRequest(
+    useCallback(async () => SystemJobTemplatesAPI.readDetail(id), [id])
+  );
+
+  useEffect(() => {
+    request();
+  }, [request, pathname]);
+
+  useEffect(() => {
+    if (!result) return;
+
+    setBreadcrumb(result);
+  }, [result, setBreadcrumb]);
+
+  const createSchedule = data =>
+    SystemJobTemplatesAPI.createSchedule(result.id, data);
+  const loadSchedules = params =>
+    SystemJobTemplatesAPI.readSchedules(result.id, params);
+  const loadScheduleOptions = () =>
+    SystemJobTemplatesAPI.readScheduleOptions(result.id);
+
+  const tabsArray = [
+    {
+      id: 99,
+      link: basePath,
+      name: (
+        <>
+          <CaretLeftIcon />
+          {i18n._(t`Back to management jobs`)}
+        </>
+      ),
+    },
+    {
+      id: 0,
+      name: i18n._(t`Schedules`),
+      link: `${match.url}/schedules`,
+    },
+  ];
+
+  if (canReadNotifications) {
+    tabsArray.push({
+      id: 1,
+      name: i18n._(t`Notifications`),
+      link: `${match.url}/notifications`,
+    });
+  }
+
+  let Tabs = <RoutedTabs tabsArray={tabsArray} />;
+  if (pathname.includes('edit') || pathname.includes('schedules/')) {
+    Tabs = null;
+  }
+
+  const LoadingScreen = (
+    <PageSection>
+      <Card>
+        {Tabs}
+        <ContentLoading />
+      </Card>
+    </PageSection>
+  );
+
+  const ErrorScreen = (
+    <PageSection>
+      <Card>
+        <ContentError error={error}>
+          {error?.response?.status === 404 && (
+            <span>
+              {i18n._(t`Management job not found.`)}
+              {''}
+              <Link to={basePath}>{i18n._(t`View all management jobs`)}</Link>
+            </span>
+          )}
+        </ContentError>
+      </Card>
+    </PageSection>
+  );
+
+  if (error) {
+    return ErrorScreen;
+  }
+
+  if (isLoading) {
+    return LoadingScreen;
+  }
```

Comment:
you might as well define these JSX blocks inside the `if` statements

e.g.
```
if (error) {
  return (<PageSection>...</PageSection>);
}
```

Reply:
done